kubernetes-azure-go/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"encoding/base64"
+
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/containerservice"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/network"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/resources"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		// Get some configuration values or set default values
+		cfg := config.New(ctx, "")
+		sshPubKey := cfg.Require("sshPubKey")
+		mgmtGroup := cfg.Require("mgmtGroupId")
```

Comment:
Noticed there's a note in the Python version that calls these out. Should we do that here and in the other templates also? (I don't have a opinion other than "consistency is nice". 😄)

```suggestion
		sshPubKey := cfg.Require("sshPubKey")
		mgmtGroup := cfg.Require("mgmtGroupId")
```

Reply:
Fixed in 965cc5b (comment added to all languages)